### ISSUE TITLE
No documentation for Julia 1.10-beta

### ISSUE THREAD

**Issue by eschnett** [JuliaLang/julia#50872]
**Describe the bug**
The web page https://docs.julialang.org/en/v1/ documents many Julia versions, including version v1.9.2 and v1.11-dev. However, there is no documentation for v1.10-beta available.

**To Reproduce**
1. Go to 'https://docs.julialang.org/en/v1/'
2. Click on the "version" drop-down menu
3. Scroll to the bottom

**Expected behavior**
Since documentation for the development version is available I expect documentation for the current beta version – which is easily available for download – do be available as well.

**Comment by mortenpi** [comment]
I think we only include RCs in the version selector?

https://github.com/JuliaLang/julia/blob/15b590bd6a1f17f3859752594d5d8ff534cd38b3/doc/make.jl#L383-L387

**Comment by mortenpi** [comment]
Yeah, probably. Just should make sure we only get one per version. What's our sequence with the pre-releases? `alpha` -> `beta` -> `rc`?

**Comment by KristofferC** [comment]
Yes

**Comment by oscardssmith** [comment]
IMO including pre-releases is probably a good idea.

**Comment by mortenpi** [comment]
I'd say this is a good, self-contained first issue for someone to hack on. As far as I can tell, it's just this function that needs to be updated:

https://github.com/JuliaLang/julia/blob/15b590bd6a1f17f3859752594d5d8ff534cd38b3/doc/make.jl#L372-L390

**Comment by Jay-sanjay** [comment]
Yes sir , that sounds interesting I would like to work on it sir.
I had one doubt that :
 function Documenter.Writers.HTMLWriter.expand_versions(dir::String, v::Versions) 
only this function needs to be updated for this issue right.....?

**Comment by Jay-sanjay** [comment]
@mortenpi  sir I have made the necessary changes in the function.

**Comment by mortenpi** [comment]
@Jay-sanjay Do you have a link to the pull request?

**Comment by Jay-sanjay** [comment]
@mortenpi  sir here it is https://github.com/Jay-sanjay/julia/pull/1

**Comment by mortenpi** [comment]
@Jay-sanjay You would need to open the PR against the JuliaLang/julia repo ([this might help if you're unsure what that means](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)).

However, as far as I can tell, your patch is not correct. I don't think it even parses. Please test it locally to make sure it works the way it should. The [contributor notes](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#improving-documentation) should offer some guidance on how to get started with that.

**Comment by abslock128w** [comment]
Hi! I’d like to work on this if it’s still relevant.

From the issue history, it looks like parts of this may already be addressed
(deployment of docs for betas/RCs, version selector fixes, etc.).
Could someone confirm whether there is any remaining work needed here,
or point me to a follow-up issue/PR where help would be useful?

Happy to take this on if guidance is available. Thanks!

**Comment by Atharvapawar1** [comment]
Hi! I'd like to work on this issue if it's still relevant. I noticed there isn't an active PR linked to it. I'll investigate the suggested changes in doc/make.jl and submit a PR if the issue is still valid.
